samples/navmodel-samples/lint-baseline.xml

```diff
@@ -1,4 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<issues format="6" by="lint 7.1.3" type="baseline" client="gradle" dependencies="false" name="AGP (7.1.3)" variant="all" version="7.1.3">
+<issues format="6" by="lint 7.2.2" type="baseline" client="gradle" dependencies="false" name="AGP (7.2.2)" variant="all" version="7.2.2">
+
+    <issue
```

Comment:
It looks like we are suppressing this in code instead now (based on all other usages of this supression): `@SuppressLint("ModifierFactoryExtensionFunction")`